core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -1011,7 +1011,9 @@ private ExecutionModel parseExplain(GenericLexer lexer, SqlExecutionContext exec
         return parseSelect(lexer);
     }
 
-    private int parseExplainOptions(GenericLexer lexer) throws SqlException {
+    private int parseExplainOptions(GenericLexer lexer, CharSequence prevTok) throws SqlException {
+        int parenPosition = lexer.getPosition();
```

Comment:
`parent`

Reply:
Actually it's the position of parenthesis after explain . Fixed name in 6e336a56 .